hack/lib/test.sh

```diff
@@ -34,7 +34,7 @@ kube::test::get_object_assert() {
 
   res=$(eval kubectl ${args} get "${kube_flags[@]}" $object -o go-template=\"$request\")
 
-  if [[ "$res" =~ ^$expected$ ]]; then
+  if [[ "$res" =~ ^$expected ]]; then
```

Comment:
Only if a test is asserting *no other content* is in the result.  I'm assuming that wouldnt be a problem really. ?  maybe it would.  im open to a better idea?